bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -130,13 +130,13 @@ export class UIEvents
         if view?
           if view.model.cursor?
             cursor = view.model.cursor()
-          if active_inspectors?
```

Comment:
I'm hoping to run all of our tests in headless chrome in near future. It shouldn't bring any performance penalty and we will have a proper testing environment. Just we need to wait for support for OSX (along with existing for Linux).